editor/configs/peopleColumns.tsx

```diff
@@ -5,20 +5,25 @@ import { PeopleListRow } from "@/types/peopleList";
 import PersonNameField from "@/components/PersonNameField";
 import { formatAddresses, formatIsoDateTime } from "@/utils/formatters";
 import { commonValidations } from "@/utils/formHelpers";
+import { compareNumbersAndBools } from "@/utils/sorting";
 
 export const ALL_PEOPLE_COLUMNS = {
-  id: { path: "id", label: "ID" },
+  id: { path: "id", label: "ID", sortable: true },
   drvr_city_name: {
     path: "drvr_city_name",
     label: "City",
+    sortable: true,
```

Comment:
I poked around the MUI DataGrid API as I was writing this and noticed that it enables sorting by default: it  uses `disableColumnSorting` setting instead. I think that's reasonable, but the column definitions in MUI include the data type, so that the grid chooses the right sorting function out-of-the-box. 

This implementation will sort values as strings by default, so maybe it's good to force the dev to enable sorting on a per-column basis so as to prompt them to consider the sort function?

Reply:
That makes sense to me to add sort config as we go!